src/components/BooksForm.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
```

Comment:
pls relocate this file into `src/containers` folder
![Screen Shot 2020-04-22 at 8 17 04 AM](https://user-images.githubusercontent.com/40712459/79951803-49a5e080-8471-11ea-890f-fffcd16765f0.png)

Reply:
Will be resolved in Milestone 3 to avoid misalignment of code